list-targets/action.yaml

```diff
@@ -39,15 +39,26 @@ runs:
       env:
         HEAD_SHA: ${{github.event.pull_request.head.sha}}
 
+    - run: go install github.com/hashicorp/terraform-config-inspect@latest
```

Comment:
We should pin the version. `latest` is unstable and insecure.
And this step requires Go, so we should clarify the requirement.

We can also install this tool by aqua.

https://github.com/aquaproj/aqua-registry/tree/main/pkgs/hashicorp/terraform-config-inspect

In that case, we need to ask users to update aqua.yaml, which isn't ideal.

Reply:
Pinned the version https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/a7d46355517b2df0a215e18c1feba58aa8e4a6f6

I did not touch aqua, since I am not familiar with aqua and you said it is not ideal.

Reply:
Golang dpendency declared in https://github.com/suzuki-shunsuke/tfaction/pull/1528/commits/1aba4757dad49d5f4f3df83a1a5205ba1d6faaad